src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml

```diff
@@ -71,8 +71,7 @@
                                 Background="CornflowerBlue" Foreground="White"
                                 Click="SaveApiKey_Click" />
                                 <TextBlock TextWrapping="Wrap" Margin="0,10,0,0">
-                            <Span>You can get or create an API key from <Hyperlink NavigateUri="https://developers.arcgis.com/api-keys/">your developer dashboard</Hyperlink>.</Span><LineBreak></LineBreak>
-                            <Hyperlink NavigateUri="https://developers.arcgis.com/net/security-and-authentication/#api-keys">More info on API Keys</Hyperlink>
+                                  <Span>You can get or create an API key as described in <Hyperlink NavigateUri="https://links.esri.com/create-an-api-key">this tutorial</Hyperlink>.</Span>
```

Comment:
You removed the more-info here and below, but maui still has the more info. Which solution are we going with?

Reply:
More-info is also not needed any longer. I missed to removed it from MAUI. Just updated the PR. Please review the latest commit.

Reply:
Same comment as above "You can create an API Key access token as described...."